src/main/kotlin/org/rust/ide/annotator/fixes/AddStructFieldsFix.kt

```diff
@@ -105,4 +106,19 @@ class AddStructFieldsFix(
 
         return null
     }
+
+    private fun specializedCreateStructLiteralField(factory: RsPsiFactory, fieldDecl: RsFieldDecl): RsStructLiteralField? {
+        val fieldTypeRef = fieldDecl.typeReference ?: return null
```

Comment:
The types are represented by `Ty` classes in our plugin. To get a `Ty` from a `typeReference`, you can use `.type` extension method. 

So this could look like

```
private val Ty.defaultValue: String = ...
```

and then 

```
fieldDecl.typeReference?.type?.defaultValue
```